src/seedu/addressbook/commands/EditCommand.java

```diff
@@ -0,0 +1,80 @@
+package seedu.addressbook.commands;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.tag.UniqueTagList;
+
+import java.util.*;
+
+public class EditCommand extends Command{
```

Comment:
Every class should have a JavaDoc header comment, so that its easy to understand the purpose of the class